tools/autograd/templates/Functions.cpp

```diff
@@ -694,31 +694,17 @@ Tensor masked_scatter_backward(const Tensor & grad, const Tensor & mask, IntArra
 
 Tensor cholesky_backward(Tensor grad, bool upper, Tensor L) {
   // cf. Iain Murray (2016); arXiv 1602.07527
+  // This gradient is symmetric, and not triangular
```

Comment:
could you add some comment here about the ambiguity and why we prefer symmetric grad?

Reply:
Sure, I will do that.